LICENSE

```diff
@@ -220,13 +220,13 @@
     3. scala-package - For details, see, scala-package/LICENSE
     4. Warp-CTC - For details, see, src/operator/contrib/ctc_include/LICENSE
     5. 3rdparty/dlpack - For details, see, 3rdparty/dlpack/LICENSE
-    6. dmlc-core - For details, see, dmlc-core/LICENSE
+    6. 3rdparty/dmlc-core - For details, see, 3rdparty/dmlc-core/LICENSE
     7. 3rdparty/mshadow - For details, see, 3rdparty/mshadow/LICENSE
-    8. nnvm/dmlc-core - For details, see, nnvm/dmlc-core/LICENSE
-    9. nnvm - For details, see, nnvm/LICENSE
-    10. nnvm-fusion - For details, see, nnvm/plugin/nnvm-fusion/LICENSE
+    8. 3rdparty/nnvm/dmlc-core - For details, see, 3rdparty/nnvm/dmlc-core/LICENSE
+    9. 3rdparty/nnvm - For details, see, 3rdparty/nnvm/LICENSE
+    10. nnvm-fusion - For details, see, 3rdparty/nnvm/plugin/nnvm-fusion/LICENSE
```

Comment:
following the same pattern, you can add the 3rdparty path for nnvm-fusion, but I guess this should also do

Reply:
Updated in https://github.com/apache/incubator-mxnet/pull/10370